backend/core/src/auth/services/authz.service.ts

```diff
@@ -38,7 +39,7 @@ export class AuthzService {
       e = await this.addUserPermissions(e, user)
 
       if (type === "user" && obj?.id && !obj?.jurisdictionId) {
-        const accessedUser = await this.userRepository.findById(obj.id)
+        const accessedUser = await findById(this.userRepository, obj.id)
```

Comment:
Suggestion (blocking): If you move `findById` to the user service and inject that (which should already inject the user repo), then this should become simplified and you won't have to pass around the user repo.